R/db.R

```diff
@@ -407,7 +407,7 @@ get_new_key <- function(dbconn, tbl_name, ids, key) {
   } else {
     shards <- shards[[1]]
   }
-  if (!DBI::dbExistsTable(dbconn, shards[1])) return(ids)
```

Comment:
I lied.  I mistook this from another refactor I was doing earlier. I'll revert this.